packages/nova-react/src/graphql/hooks.ts

```diff
@@ -153,13 +166,42 @@ export function useFragment<TKey extends KeyType>(
  *                    fragment.
  * @returns The data corresponding to the field selections and a function to perform the refetch.
  */
+export function useRefetchableFragment<
+  TQuery extends OperationType,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _TKey extends KeyType,
```

Comment:
This one could do with a comment perhaps, but not necessary if it's captured as a test that would fail without this type parameter definition missing.

Reply:
Added a test :)